### DISCUSSION TITLE
Sampling subgraph/neighborhood/k-hop from node index in hetero graph

### DISCUSSION THREAD

**Discussion by AdrianBZG** [pyg-team/pytorch_geometric#4331] [Q&A]
Hi,

Is it possible to sample a subgraph or build the neighborhood/k-hop subgraph from a given node in a hetero graph?

Thanks.

**Comment by rusty1s** [comment]
Our `NeighborLoader` class can well handle the sampling on heterogeneous graphs. In addition, we are working on a `HeteroData.subgraph()` implementation. Its progress is tracked here: https://github.com/pyg-team/pytorch_geometric/issues/4001
